src/ik_parallel.h

```diff
@@ -195,84 +221,84 @@ struct IKParallel
         BLOCKPROFILER("solve mt");
 
         // prepare
-        iteration_count = 0;
-        result = problem.initial_guess;
-        timeout = problem.timeout;
-        success = false;
-        finished = 0;
-        for(auto& s : solver_solutions)
-            s = problem.initial_guess;
-        for(auto& s : solver_temps)
-            s = problem.initial_guess;
-        for(auto& s : solver_success)
+        iteration_count_ = 0;
+        result_ = problem_.initial_guess;
+        timeout_ = problem_.timeout;
+        success_ = false;
+        finished_ = 0;
+        for(auto& s : solver_solutions_)
+            s = problem_.initial_guess;
+        for(auto& s : solver_temps_)
+            s = problem_.initial_guess;
+        for(auto& s : solver_success_)
             s = 0;
-        for(auto& f : solver_fitness)
+        for(auto& f : solver_fitness_)
             f = DBL_MAX;
-        for(auto& s : solvers)
-            s->canceled = false;
+        for(auto& s : solvers_)
+            s->canceled_ = false;
 
-        // run solvers
+        // run solvers_
```

Comment:
`solvers` isn't referring to the member variable in this instance

Reply:
Addressed in [e9c01ee](https://github.com/PickNikRobotics/bio_ik/pull/1/commits/e9c01ee4ef00ced5d79bb788ab01a067e93e922c)